code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -227,25 +261,72 @@ Due to keyboard shortcuts, the second one is not necessarily the remote eye's lo
 		return
 	food.apply_status_effect(/datum/status_effect/slime_food, user)
 
-	monkeys--
-	monkeys = round(monkeys, 0.1) //Prevents rounding errors
+	stored_monkeys--
+	stored_monkeys = round(stored_monkeys, 0.1) //Prevents rounding errors
 	spit_out(food, target_turf)
-	xeno_hud.on_update_hud(LAZYLEN(stored_slimes), monkeys, max_slimes)
+	xeno_hud.on_update_hud(LAZYLEN(stored_slimes), stored_monkeys, max_slimes)
 
-///Recycles the target monkey
-/obj/machinery/computer/camera_advanced/xenobio/proc/monkey_recycle(mob/living/user, mob/living/target_mob)
-	if(!ismonkey(target_mob))
-		return
-	if(!target_mob.stat)
+/// Check whether we can recycle monkeys at all. Optionally, displays a balloon alert over a target atom for feedback.
+/obj/machinery/computer/camera_advanced/xenobio/proc/can_recycle_monkeys(mob/living/user, atom/target_atom)
+	PRIVATE_PROC(TRUE)
+	var/obj/machinery/monkey_recycler/connected_recycler = connected_recycler_ref?.resolve()
+	if(isnull(connected_recycler))
+		to_chat(user, span_warning("There is no connected monkey recycler. Use a multitool to link one."))
+		if(target_atom)
+			target_atom.balloon_alert(user, "no recycler linked!")
+		return FALSE
+	return TRUE
+
+/// Check whether we can recycle the target monkey. Optionally takes in a user to display errors to.
+/obj/machinery/computer/camera_advanced/xenobio/proc/can_recycle_target_monkey(mob/living/carbon/human/target_human, mob/living/user)
+	PRIVATE_PROC(TRUE)
+	if(!ismonkey(target_human))
+		if(user)
+			target_human.balloon_alert(user, "not a monkey!")
+		return FALSE
+	if(target_human.stat < DEAD)
+		if(user)
+			target_human.balloon_alert(user, "isn't dead!")
```

Comment:
Small thing but I'd probably prefer `not dead!` / `it's not dead!` over `isn't`
Solely for the fact that the lazarus injector uses `it's not dead` and I like consistency